_includes/macros.html

```diff
@@ -80,20 +82,25 @@
 
    ========================================================================== #}
 
-{% macro share(title, hide_heading=false, twitter_options={}) %}
-    <aside>
-        <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">Share this page</h1>
+{% macro share(title, hide_heading=false, twitter_options={}, share_options={}) %}
+    {% set text = share_options.text | default('Share this page') %}
+    {% set additional_classes = share_options.additional_classes  %}
+
+    <aside {{ ('class=' ~ additional_classes) if additional_classes }}>
+        <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">{{ text }}</h1>
         <ul class="post_share list__horizontal">
             <li class="list_item">
                 <a class="share-icon"
-                   href="http://api.addthis.com/oexchange/0.8/forward/email/offer?url={{ request.url|urlencode }}&amp;title={{ title }}&amp;pubid=ra-4da354ee346886d2">
+                   href="http://api.addthis.com/oexchange/0.8/forward/email/offer?url={{ request.url|urlencode }}&amp;title={{ title }}&amp;pubid=ra-4da354ee346886d2"
+                   target="_blank">
                     <span class="cf-icon cf-icon-email-social-square"></span>
                     <span class="u-visually-hidden">Share by email</span>
                 </a>
             </li>
             <li class="list_item">
                 <a class="share-icon"
-                   href="https://www.facebook.com/dialog/share?app_id=210516218981921&amp;display=page&amp;href={{ request.url|urlencode }}&amp;redirect_uri={{ request.url|urlencode }}">
+                   href="https://www.facebook.com/dialog/share?app_id=210516218981921&amp;display=page&amp;href={{ request.url|urlencode }}&amp;redirect_uri={{ request.url|urlencode }}"
```

Comment:
The `app_id` for Facebook here and the `pubid` in the addthis.com URL earlier are specific to cfgov-refresh right? It probably would make sense to pull those out into a config settings file (along with other config settings such as Google Analytics, etc.). Certainly doesn't have to happen in this PR, but just making a note.